pkg/domain/infra/abi/pods_stats.go

```diff
@@ -0,0 +1,85 @@
+package abi
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/libpod"
+	"github.com/containers/libpod/pkg/cgroups"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/docker/go-units"
+	"github.com/pkg/errors"
+)
+
+// PodStats implements printing stats about pods.
+func (ic *ContainerEngine) PodStats(ctx context.Context, namesOrIds []string, options entities.PodStatsOptions) ([]*entities.PodStatsReport, error) {
+	// Cgroups v2 check for rootless.
+	if rootless.IsRootless() {
+		unified, err := cgroups.IsCgroup2UnifiedMode()
+		if err != nil {
+			return nil, err
+		}
+		if !unified {
+			return nil, errors.New("pod stats is not supported in rootless mode without cgroups v2")
```

Comment:
This should be in  a man page somewhere if it is not already.

Reply:
Great catch, added!